packages/core/react-loosely-lazy/__tests__/integration.test.tsx

```diff
@@ -395,6 +395,7 @@ describe('renders', () => {
   });
 });
 
+/* eslint-disable jest/no-commented-out-tests */
```

Comment:
/nit

Why not skip instead? Actually, why keep this here still?

Reply:
I don't really know what the intention of the test was, since it wasn't written by me. @albertogasparin care to shed some light? Otherwise I will dig into the history.

Reply:
Probably placeholders for tests that I wanted to add (and never did). If we cover such scenarios somewhere else, happy for them to be deleted